<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Z2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>791</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="mainMenu">
     <widget class="QPushButton" name="btnSingle">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>100</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>单人游戏</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnMultiple">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>170</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>局域网联机</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblZ2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>161</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sylfaen</family>
        <pointsize>72</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Z2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSetting">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>310</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>载入游戏</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnJoin">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>240</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>加入游戏</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="lobby">
     <widget class="QPushButton" name="btnStartGame">
      <property name="geometry">
       <rect>
        <x>609</x>
        <y>469</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>开始游戏</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsMapDisplay">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>40</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>地图:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>0</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>地图预览</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>0</y>
        <width>20</width>
        <height>251</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QComboBox" name="cmbMap">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>221</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
     <widget class="QTableWidget" name="tablePlayer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>551</width>
        <height>241</height>
       </rect>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>新建行</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>序号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>组</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>位置</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>颜色</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>难度</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>490</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblMapName">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>0</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>469</x>
        <y>-1</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>载入存档</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="loadingPage">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>500</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>正在加载游戏...</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>服务器地址：</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblAddress">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>140</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>127.0.0.1:12345</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>119</x>
        <y>189</y>
        <width>391</width>
        <height>221</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCancel">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>480</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="connectPage">
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>229</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textInputAddress">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>219</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>221</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>地址</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>加入局域网游戏</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="textInputId">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>259</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>261</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>玩家id</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>490</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
